DOCS-3912: Add sub-part frame note (#4357)
viamrobotics/docs: docs/operate/reference/architecture/parts.md

{{<imgproc src="/build/configure/parts/part-mgmt.png" resize="x1100" declaredimensions=true alt="The Viam app interface with the part actions dropdown open. Options include rename, restart part, make main part, view setup instructions, view history, and delete part." style="width:500px" class="shadow" >}}

> By default, all sub-parts are added to the [frame system](/operate/reference/services/frame-system/) at the world origin.

Reviewer: Just to avoid passive voice - doesn't really improve it so either way
```suggestion
By default, all sub-parts appear in the [frame system](/operate/reference/services/frame-system/) at the world origin.
```

> You can specify translations for sub-parts by configuring their frames with appropriate translation values relative to their parent frame.

Author: ```suggestion
You can specify translations for sub-parts by configuring their frames with appropriate translation values relative to their parent frame.
You can view the `frame` of each sub-part in the `remotes` section of the debug config.
```
